src/Tabs.scss

```diff
@@ -0,0 +1,33 @@
+.Tabs {
+  &__TabsList {
+    display: flex;
+    gap: 10px;
+
+    list-style: none;
+
+    width: 300px;
+
+    border-bottom: 1px solid grey;
+  }
+
+  &__button {
+    border: none;
+    background-color: burlywood;
+    padding: 5px;
+
+    margin-bottom: -1px;
+
+    &--active {
+      background-color: rgb(230, 160, 69);
+
+      border: solid grey;
+      border-width: 1px 1px 0 1px;
+      border-bottom: none;
+      border-radius: 5px 5px 0 0;
+    }
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
```

Comment:
Please put an empty line in the end of the file

Reply:
Agree.